pyvista/plotting/renderers.py

```diff
@@ -336,12 +336,12 @@ def active_renderer(self):  # numpydoc ignore=RT01
         return self._renderers[self._active_index]
 
     @property
-    def shape(self):  # numpydoc ignore=RT01
+    def shape(self) -> tuple[int, int]:  # numpydoc ignore=RT01
```

Comment:
On the first code path we have `self._shape = (n + m,)` so I think we need
```suggestion
    def shape(self) -> tuple[int] | tuple[int, int]:  # numpydoc ignore=RT01
```
(and same in the docstring).

And why do we need `RT01`? [Isn't that "No returns section found"](https://numpydoc.readthedocs.io/en/v1.6.0rc1/validation.html#built-in-validation-checks)?

Reply:
I was just leaving things as they were, but since we are touching these, I cleanup up.  I removed any ones I could find touched.

Reply:
Also added to the returns sections